spec/post_spec.rb

```diff
@@ -13,15 +19,16 @@
     expect(post).to_not be_valid
   end
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍